### ISSUE TITLE
[1.16] Crash when adding a resource pack

### ISSUE THREAD

**Issue by embeddedt** [Pyre540/tinkers-levelling-addon#7]
When adding a resource pack on the main menu (before loading a world), the following exception is raised and the game eventually crashes:

```
java.lang.IllegalStateException: Tag tconstruct:modifiable used before it was bound
	at net.minecraft.tags.TagRegistry$NamedTag.func_232944_c_(TagRegistry.java:131)
	at net.minecraft.tags.TagRegistry$NamedTag.func_230235_a_(TagRegistry.java:142)
	at slimeknights.tconstruct.library.tools.helper.ModifierUtil.getModifierLevel(ModifierUtil.java:193)
	at pyre.tinkerslevellingaddon.setup.ClientEventHandler.onTooltipEvent(ClientEventHandler.java:66)
	at net.minecraftforge.eventbus.ASMEventHandler_15_ClientEventHandler_onTooltipEvent_ItemTooltipEvent.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	at net.minecraftforge.event.ForgeEventFactory.onItemTooltip(ForgeEventFactory.java:275)
	at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:701)
	at net.minecraft.client.Minecraft.func_213251_b(Minecraft.java:642)
	at net.minecraft.client.util.SearchTree.func_194042_b(SourceFile:36)
	at net.minecraft.client.util.SearchTreeReloadable.func_194040_a(SourceFile:37)
	at net.minecraft.client.util.SearchTree.func_194040_a(SourceFile:29)
	at net.minecraft.client.util.SearchTreeManager.func_195410_a(SearchTreeManager.java:21)
	at net.minecraft.resources.IResourceManagerReloadListener.func_232635_a_(IResourceManagerReloadListener.java:18)
```

I believe this issue occurs because it is not legal to access tags when a world is not loaded. However, it seems that it is possible for `ItemTooltipEvent` to be fired without a world being present.

I would guess that this also affects 1.18, but I'm not 100% sure.

**Comment by Pyre540** [comment]
It seems it only affects 1.16. From my testing, 1.18 is working fine. Either way I will probably make changes in both versions to be extra safe. 
Thanks for reporting :)

**Comment by thebikwirm** [comment]
when will this be fixed in 16.5

**Comment by Pyre540** [comment]
1.16.5 and 1.18.2 update is out.
